docs/source/inference.rst

```diff
@@ -310,3 +310,38 @@ The output files will be in the ``mapillary_inference_output`` folder, with:
       -  ``img_0_y_0.png``: the original image resized after conditioning image insertion
 
       -  ``img_0_y_t.png``: the noisy image given to the model
+
+******************************************************
+ Generate a video with diffusion model for inpainting
+******************************************************
+
+Download the testdataset & pretrained model
+=====================================
+
+.. code:: bash
+
+   wget https://www.joligen.com/models/mario_vid.zip
+   unzip mario_vid.zip -d checkpoints
+   rm mario_vid.zip
```

Comment:
model is here: juliew@neptune7:/data1/juliew/continental/PR_joliGEN/checkpoints/mario_vid